lib/context.js

```diff
@@ -50,22 +74,24 @@ const proto = module.exports = {
       res: '<original node res>',
       socket: '<original node socket>'
     };
-  },
+  }
 
   /**
    * Similar to .throw(), adds assertion.
    *
-   *    this.assert(this.user, 401, 'Please login!');
+   *    ctx.assert(ctx.user, 401, 'Please login!');
    *
    * See: https://github.com/jshttp/http-assert
    *
-   * @param {Mixed} test
+   * @param {Mixed} value
    * @param {Number} status
    * @param {String} message
+   * @param {Mixed} opts
    * @api public
    */
-
-  assert: httpAssert,
+  assert(value, status, msg, opts) {
```

Comment:
Why the attempt to save bytes? Using `.apply` or any other method takes away argument clarity and is, in fact, slower anyway.